### ISSUE TITLE
[FEATURE]: Exclude empty aria-label

### ISSUE THREAD

**Issue by projct1** [jackdomleo7/Cooltipz.css#64]
Will be good, if [aria-label] selector ignoring empty attributes like this: ```[aria-label]:not([aria-label]="")```
Now i have to use crutch:
```css
[aria-label=""]::before, [aria-label=""]::after {
  display: none;
}
```

**Comment by jackdomleo7** [comment]
Good idea

**Comment by jackdomleo7** [comment]
Look out for this in a future release.

**Comment by jackdomleo7** [comment]
Released in [v2.1.0](https://github.com/jackdomleo7/Cooltipz.css/releases/tag/v2.1.0)
